content/docs/install/helm-install/index.md

```diff
@@ -222,12 +222,12 @@ in the [helm-charts](https://github.com/keptn/keptn/tree/master/installer/manife
 * The **Control Plane with the Execution Plane (for Continuous Delivery)**
 can be installed by the following command:
 ```
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh --set=continuousDelivery.enabled=true
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh --set=bridge.installationType=CONTINUOUS_DELIVERY
```

Comment:
This setting is actually in the default values anyways, so there's no need to set anything here. We can just remove that `--set`
```suggestion
helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh
```

Reply:
@mowies the heading explicitly states `The **Control Plane with the Execution Plane (for Continuous Delivery)` , would it make sense to keep it here, otherwise these installation instructions or generic ones wouldn't differ and could cause confusion as to why the same instructions under a different heading:

<img width="1140" alt="image" src="https://user-images.githubusercontent.com/9951135/209889523-cd741345-8533-4d9a-ba45-a9f59c091bd6.png">

or i can remove the `--set=bridge.installationType=CONTINUOUS_DELIVERY` command and remove the `for Continuous Delivery` from the headline.

Reply:
Ok fair point. Then I would say let's also remove the mention of continuous delivery here. That paragraph just talks about how to set Helm values in a generic way anyways. We don't need super specific examples. I would change the paragraph to this:


> * The **Control Plane** and a `LoadBalancer` for exposing Keptn can be installed by the following command:
> ```
> helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh --set=apiGatewayNginx.type=LoadBalancer
> ```

So let's just remove the first example without the load balancer and just put in the example with the load balancer setting. But let's also remove the continuous delivery setting.

Reply:
apologies for late response, I have addressed this and pushed it, please review again.